### DISCUSSION TITLE
Is there a portable version?

### DISCUSSION THREAD

**Discussion by artad-sys** [valinet/ExplorerPatcher#1586] [Q&A]
Hi!

My company unfortunately upgraded machines to Windows 11 and of course I lost quick launch ang ungroup taskbar icons options.
I'm not a local admin on my machine, and I don't want to be.
Is there an option to use EP without any local admin permissions?
Can somehow a portable version made?

Any ideas please?

Thank you!

**Comment by pyrates999** [comment]
If you can't change it as a local admin, then explorer patcher would be prevented even if they tried by the OS.
